engine/src/main/java/org/hibernate/validator/internal/SuppressForbidden.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to be used for suppression of forbidden apis errors.
+ */
+@Retention(RetentionPolicy.CLASS)
```

Comment:
Not sure why it cannot be SOURCE, but I've checked here:
https://github.com/policeman-tools/forbidden-apis/blob/master/src/main/java/de/thetaphi/forbiddenapis/SuppressForbidden.java
and it has `RetentionPolicy.CLASS` for it...